packages/health-check/src/checks/critical.js

```diff
@@ -60,7 +86,44 @@ async function uploadCheck(done) {
     data.ip = error?.response?.ip ?? null;
   }
 
-  done({ name: "upload_file", time: calculateElapsedTime(time), ...data });
+  done({ name, time: calculateElapsedTime(time), ...data });
+}
+
+// skylinkHealthCheck checks if the skylink has reached full redundancy
+async function skylinkHealthCheck(skylink, numRetries = 30, authCookie, isLarge = false) {
+  // Get the health of the skylink
+  const response = await got(`https://${process.env.PORTAL_DOMAIN}/skynet/health/skylink/${skylink}`, {
+    headers: { cookie: authCookie },
+  });
+  const healthData = getResponseContent(response);
+
+  // Check Basesectorredundancy first
+  if (healthData.basesectorredundancy !== 10 && numRetries > 0) {
+    // Semi-smart sleep before retrying. Sleep longer if the redundancy is
+    // lower.
+    await new Promise((r) => setTimeout(r, (10 - healthData.basesectorredundancy) * 1000));
+    return skylinkHealthCheck(skylink, numRetries - 1, authCookie, isLarge);
+  }
+
+  // Check the Fanout redundancy if it is a large file
+  if (isLarge && healthData.fanoutredundancy != 3 && numRetries > 0) {
+    // Semi-smart sleep before retrying. Sleep longer if the redundancy is
+    // lower.
+    await new Promise((r) => setTimeout(r, (3 - healthData.fanoutredundancy) * 10000));
```

Comment:
did you notice the `10_000` here? maybe extract it into a `SECOND` const

Reply:
I intentionally made it 10,000 since it was going to take longer.